risc0/zkvm/platform/src/syscall.rs

```diff
@@ -71,6 +71,9 @@ pub mod reg_abi {
 
 pub const DIGEST_WORDS: usize = 8;
 pub const DIGEST_BYTES: usize = WORD_SIZE * DIGEST_WORDS;
+pub const MAX_BUF_BYTES: usize = 64 * 1024;
+pub const MAX_BUF_WORDS: usize = MAX_BUF_BYTES / WORD_SIZE;
+pub const MAX_SHA_COMPRESS_ROUNDS: usize = 1000;
```

Comment:
`MAX_SHA_COMPRESS_ROUNDS` might be misread to imply that it controls the number of rounds within the compression function.
```suggestion
pub const MAX_SHA_COMPRESS_BLOCKS: usize = 1000;
```

Reply:
WDYT of `MAX_SHA_COMPRESS_COUNT`? I can also add a comment